README.md

```diff
@@ -22,9 +27,76 @@ Code quality checking and enforcement is done with the following tools:
 
 Error and exception logging is done using Sentry.
 
-## Deployment
- - Provide `SENTRY_DSN` and `SENTRY_ENVIRONMENT` as environmental variables
-   during deployment.
+#### Environmental Variables
+
+| Name                                  | Description                                  | Default   |
+|---------------------------------------|----------------------------------------------|-----------|
+| DB_HOST                               | The MongoDB host to connect to.              | localhost |
+| DB_PORT                               | The port to connect to MongoDB on.           | 27017     |
+| DB_USER                               | The username to access the MongoDB instance. | admin     |
+| DB_NAME                               | The name of the MongoDB instance.            | forms     |
+| DB_PASSWORD                           | The password to access the MongoDB instance. | pwd       |
+| SENTRY_DSN                            | A Sentry error monitoring Data Source Name.  |           |
+| SENTRY_ENVIRONMENT                    | A Sentry error monitoring DSN environment.   | local     |
+| FEATURE_FORMR_PARTB_COVID_DECLARATION |                                              | true      |
```

Comment:
Useful to include a description of this, i.e. that it is a feature flag that controls whether to include the covid declaration section in FormR PartB